kong/dao/migrations/cassandra.lua

```diff
@@ -423,4 +423,27 @@ return {
     end,
     down = function(_, _, dao) end
   },
+  {
+    name = "2017-07-31-113200_jwt_preflight_default",
+    up = function(_, _, dao)
+      local rows, err = dao.db:query([[
+        SELECT * FROM plugins WHERE name = 'jwt';
```

Comment:
The default page size for the driver is (currently) 1000. However, if the page size ever changes, or becomes configurable, or if such an entity has more than a 1000 rows, or simply because we wish to enforce best C* practices to avoid spreading bad code snippets by copy/paste, then we should use the proper way of aggregating all rows:

```lua
for rows, err in cluster:iterate("SELECT * FROM plugins WHERE name = 'jwt'") do
  if err then
    return err
  end
  
  if row.config.authenticate_preflight == nil then
    -- do update
  end
end
```